modules/sbt-plugin/src/main/scala/ch/epfl/scala/debugadapter/sbtplugin/DebugAdapterPlugin.scala

```diff
@@ -100,7 +100,12 @@ object DebugAdapterPlugin extends sbt.AutoPlugin {
   def runSettings: Seq[Def.Setting[_]] = Seq(
     startMainClassDebugSession := mainClassSessionTask.evaluated,
     startRemoteDebugSession := remoteSessionTask.evaluated,
-    stopDebugSession := stopSessionTask.value
+    stopDebugSession := stopSessionTask.value,
+    Keys.compile / Keys.javacOptions := {
+      val jo = (Keys.compile / Keys.javacOptions).value
+      if (jo.contains("-g")) jo
```

Comment:
According to the `javac` documentation:

> -g
Generates all debugging information, including local variables. By default, only line number and source file information is generated.
-g:[lines, vars, source]
Generates only the kinds of debugging information specified by the comma-separated list of keywords. Valid keywords are:
lines
Line number debugging information.
vars
Local variable debugging information.
source
Source file debugging information.
-g:none
Does not generate debugging information.

So if it contains any of `-g:foo, bar` we should probably not add `-g`.

```suggestion
      if (jo.contains("-g")) jo
```